build.py

```diff
@@ -1708,6 +1714,30 @@ def tensorrtllm_prebuild(cmake_script):
     # Export the TRT_ROOT environment variable
     cmake_script.cmd("export TRT_ROOT=/usr/local/tensorrt")
     cmake_script.cmd("export ARCH=$(uname -m)")
+    cmake_script.cmd(
+        'export LD_LIBRARY_PATH="/usr/local/cuda/compat/lib.real:${LD_LIBRARY_PATH}"'
+    )
+
+
+def tensorrtllm_postbuild(cmake_script, repo_install_dir, tensorrtllm_be_dir):
+    # TODO: Update the CMakeLists.txt of TRT-LLM backend to install the artifacts to the correct location
+    cmake_destination_dir = os.path.join(repo_install_dir, "backends/tensorrtllm")
+    cmake_script.mkdir(cmake_destination_dir)
+    # Copy over the TRT-LLM wheel for later installation
+    cmake_script.cp(
+        os.path.join(tensorrtllm_be_dir, "tensorrt_llm", "build", "tensorrt_llm-*.whl"),
+        cmake_destination_dir,
+    )
+
+    # Copy over the TRT-LLM backend libraries
+    cmake_script.cp(
+        os.path.join(tensorrtllm_be_dir, "build", "libtriton_tensorrtllm*.so"),
+        cmake_destination_dir,
+    )
+    cmake_script.cp(
+        os.path.join(tensorrtllm_be_dir, "build", "triton_tensorrtllm_worker"),
```

Comment:
is this a correct value, i thought target file will be named as `triton_tensorrtllm_worker.so` ?

Reply:
Yes, this should be correct. The `triton_tensorrtllm_worker` is not a .so:
```
root@sjc-dc2-dgx1-7:/opt/tritonserver/tensorrtllm_backend/ci/L0_backend_trtllm# ls -l /opt/tritonserver/backends/tensorrtllm/
total 604
-rwxrwxrwx 1 triton-server triton-server  96928 Apr 12 19:10 libtriton_tensorrtllm.so
-rwxrwxrwx 1 triton-server triton-server 490168 Apr 12 19:10 libtriton_tensorrtllm_common.so
-rwxrwxrwx 1 triton-server triton-server  28432 Apr 12 19:10 triton_tensorrtllm_worker
```